onnx/defs/math/defs.cc

```diff
@@ -95,6 +95,50 @@ ONNX_OPERATOR_SET_SCHEMA(
     7,
     OpSchema().FillUsing(MathDocGenerator("subtraction")));
 
+static const char* Mod_doc = R"DOC(
+  Performs element-wise binary modulus (with Numpy-style broadcasting support). 
+  The sign of the remainder is the same as that of the Divisor.
+
+  Mod operator can also behave like C fmod() or numpy.fmod . In this case the remainder will be 
+  floating point.  The sign of the remainder however, will be the same as the Dividend 
```

Comment:
What is the type of the returned value if "fmod" is specified? The op-schema type indicates that the type is the same, regardless of fmod. In this case, the line "the remainder will be floating point" seems wrong and should be removed.

Reply:
Removed the erroneous documentation line